src/services/structures/structures.class.js

```diff
@@ -166,14 +166,17 @@ exports.Structures = class Structures extends Service {
       }
 
       //User Filters
-      let { pix, diplome } = req.query;
+      let { pix, diplome, cv } = req.query;
       if (pix !== undefined) {
         pix = pix.split(',').map(k => parseInt(k));
         queryFilter['conseillerObj.pix.palier'] = { $in: pix };
       }
       if (diplome !== undefined) {
         queryFilter['conseillerObj.estDiplomeMedNum'] = (diplome === 'true');
       }
+      if (cv !== undefined) {
+        queryFilter['conseillerObj.cv'] = (cv === 'true') ? { '$ne': null } : { $in: [null, undefined] };
```

Comment:
pourquoi ne pas utiliser `{ $eq: null } `au lieu de `{ $in: [null, undefined] }` ?

Reply:
parce que je me suis pris une erreur "Invalid query parameter" en le faisant et avant cela j'avais testé $exists qui ne fonctionnait pas non plus :/

Reply:
Par contre je peux me contenter de { $in: [null] }